packages/cli/src/lib.rs

```diff
@@ -279,6 +286,44 @@ where
     // Render the template
     template.render(&target_dir, pkg_manager, &package_name, alpha, mobile)?;
 
+    // Install dependencies
+
+    if install_deps {
+        if pkg_manager.is_node() {
+            if cfg!(target_os = "windows") {
+                let install_cmd_output = Command::new("cmd")
+                    .args(&["/C", pkg_manager.install_cmd().unwrap()])
+                    .spawn()
+                    .unwrap()
+                    .wait_with_output()
+                    .unwrap();
+
+                if !install_cmd_output.status.success() {
+                    eprintln!(
+                        "{BOLD}{RED}✘{RESET} Failed to install dependencies via `{}`",
+                        pkg_manager.install_cmd().unwrap()
+                    );
+                    exit(1);
```

Comment:
This should be a yellow info type of message and not a hard error so don't exist the tool and proceed with the instruction as if we never tried to install deps, i.e. print the install command and the dev command.